macros/tests.sql

```diff
@@ -150,9 +150,15 @@
     {{ dbt_utils.except() }}
     select '-' as diff, {{columns}} from actual)
 
-    select * from extra_entries
-    UNION ALL
-    select * from missing_entries
+    {% if options.get("primary_key", '') | length > 0 %}
```

Comment:
UNION ALL can have an order by that works on the union, then we can simplify the code: 

```
    {% set sort_field = options.get("primary_key") %}
    select * from extra_entries
    UNION ALL
    select * from missing_entries
    {% if sort_field %}
      order by {{ sort_field }}
    {% endif %}
```

It works in both Snowflake, BigQuery and Postgres.

Reply:
Hi
Agree that `primary_key` is not the best option
What about `output_sort_field`?

As for UNION ALL ORDER BY - Yes, great suggestion - didn't know it :) It even works in Athena I use as a primary connector

Reply:
I like the name, thank you!